waiter/config-composite.edn

```diff
@@ -34,9 +34,20 @@
 
  ; ---------- Security ----------
 
- :authenticator-config {:kind :one-user
+ :authenticator-config {:kind :composite
+                        :composite {:factory-fn waiter.auth.composite/composite-authenticator
+                                    :authentication-schemes {:one-user {:authenticator-factory-fn waiter.auth.authentication/one-user-authenticator
```

Comment:
We should continue to use `:factory-fn` instead of introducing `:authenticator-factory-fn` for consistency on how we initialize components.

Reply:
Can we rename `authentication-schemes` as `auth-providers`?

Reply:
done